proprietary/tokens/src/common/ams/typography.tokens.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ams": {
+    "common": {
+      "typography": {
+        "body-text": {
+          "font-family": { "value": "{ams.brand.typography.font-family.body-text}" },
+          "font-size": { "value": "{ams.brand.typography.font-size.md}" },
```

Comment:
This and line height apply to `md` – I chose not to make that level explicit, but to make these values the baseline, overridden by the variants, just like we do in CSS.